guides/_subsections/common/navigate-to-topology.md

```diff
@@ -1,9 +1,9 @@
 {% capture navigate-to-topology-include-text %}
-  When you need to view or perform actions on multiple target JVMs at the same time, <b>Topology View</b> is the way.
+  When you need to view or perform actions on multiple target <b>JVMs</b> at the same time, <i>Topology View</i> is the way.
 {% endcapture %}
 {% include howto_step.html
-  summary="Navigate to Topology"
+  summary="Navigate to <i>Topology</i>"
   image-name="2.3.0/navigate-to-topology.png"
-  caption="Use the bottom control bar to adjust the graph view as needed."
+  caption="Use the bottom control bar to adjust the <i>Graph View</i> as needed."
```

Comment:
should "control bar" be italicized? since it's something the user interacts with like a button/label?